taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTargetView.java

```diff
@@ -487,6 +536,46 @@ public void onClick(View v) {
             (int) lastTouchX, (int) lastTouchY);
         final boolean clickedInsideOfOuterCircle = distanceToOuterCircleCenter <= outerCircleRadius;
 
+        if (target.showSequencePagination && clickedInsideOfOuterCircle) {
+          if (target.sequenceCurrentTargetIndex < target.sequenceTargetCount) {
+
+            int buttonHeight;
+            if (skipLayout != null && nextLayout != null) {
+              buttonHeight = Math.max(skipLayout.getHeight(), nextLayout.getHeight());
+            } else if (skipLayout != null) {
+              buttonHeight = skipLayout.getHeight();
+            } else {
+              buttonHeight = nextLayout.getHeight();
```

Comment:
If both `skipLayout` and `nextLayout` are null, it will cause an exception, unless you are sure this condition will never happen.

Reply:
You are right, it should have another check here...